docs/inquiry-portal/404-inquiries.md

```diff
@@ -7,7 +7,7 @@ This section contains anonymized feedback and inquiries, related to the API Cond
     The date headers provide important context as some information may have changed since the time of an inquiry and response.
 
 ## Paragraph (a)(3)(iv): API Fees - Permitted Fee (Value-Added Services)
-### July 2021

```

Comment:
@imkacarlson

Reply:
See main diff for edits since I made two commits for the inquiry pages